### DISCUSSION TITLE
Some problems about OLE objects and Chinese font

### DISCUSSION THREAD

**Discussion by guoyongwei** [iTwin/itwinjs-core#7258] [General]
itwin synchronizer does not recognize OLE objects, these objects are copied from OFFICE  software, is there a solution for this now?
The Chinese font of the text in the custom object can not be displayed normally, and the Chinese font can be displayed normally after blowing up. Is there a solution for this now?

**Comment by bbastings** [comment]
Linked/Embedded OLE is not currently supported.

**Reply by guoyongwei** [reply]
Do you have any plan to support this kind of OLE objects in the .dwg file? Because this is a common usage.

**Comment by bbastings** [comment]
There are no plans at this time.
